### PR TITLE
[opt](session variable) max_msg_size_of_result_receiver

### PR DESCRIPTION
**Pull request by zhiqiang-hhhh**
fix `[Err] 1105 - TTransportException, msg: MaxMessageSize reached`

MaxMessageSize is used to limit the max size of one filed of thrift struct during deserialization, default value of MaxMessageSize is about 104MB,  eg. `set max_msg_size_of_result_receiver = 204000000` can increase MaxMessageSize to 204 MB

### CODE REVIEW

**Comment by zhiqiang-hhhh** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/qe/SessionVariable.java [hunk 1]
```diff
@@ -515,6 +516,8 @@ public class SessionVariable implements Serializable, Writable {
 
     public static final String SHOW_ALL_FE_CONNECTION = "show_all_fe_connection";
 
+    public static final String MAX_MSG_SIZE_OF_RESULT_RECEIVER = "max_msg_size_of_result_receiver";
```
Comment (yiguolei) [inline fe/fe-core/src/main/java/org/apache/doris/qe/SessionVariable.java:519]: change variable name to   max_thrift_message_size

**Review by yiguolei — DISMISSED** [review]

**Comment by zhiqiang-hhhh** [comment]
run buildall

**Review by yiguolei — APPROVED** [review]

**Review by yiguolei — APPROVED** [review]

**Review by wm1581066 — APPROVED** [review]
LGTM

**Pull request merged**